bindings/python/iota_sdk/types/committee.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from dataclasses import dataclass, field
+from dataclasses_json import config
+from iota_sdk.types.common import HexStr, json
+
+
+@json
+@dataclass
+class CommitteeMember:
```

Comment:
Maybe a leftover, I don't think this deserves a files, I'd move it with `CommitteeResponse`